src/oop/object_independence/hard/q4/QuestionText.md

```diff
@@ -0,0 +1,49 @@
+# oop-object_independence-hard-q4
+
+- Java question
+
+## Question Text
+
+Wear and tear happens to things that get used over a long period of time. Suppose you ran a museum, and you wanted to 
+keep track of the quality of several `Painting` objects. Using the requirements below, create two classes, one called 
+`Painting` to model the game and one called `TestMuseum` to use it.
+
+## Painting Class
+
+Your `Painting` class must have the following items:
+
+- Attributes:
+    - A `String` attribute called **name**
+    - A `String` attribute called **quality**
+    - Note that both of these attributes should not be accessible from another class
+
+- Constructor:
+    - A constructor visible from another class which accepts the parameters `String, String` and sets the **name**
+      and **quality** attributes with the passed values.
+    - This constructor should be visible to other classes
+
+- Methods (all the following methods should be visible from another class):
+    - A method called `getName` that returns the value of the **name** attribute
+    - A method called `getQuality` that returns the value of the **quality** attribute
+    - A method called `setName` that accepts a `String` as a parameter, updates the **name** attribute to the passed 
```

Comment:
Please add the return type for the set methods.

Reply:
It already does? Unless "and returns nothing" isn't clear?